config/config.go

```diff
@@ -715,6 +715,10 @@ type GeoPackageCommon struct {
 	// +optional
 	Fid string `yaml:"fid,omitempty" json:"fid,omitempty" validate:"required" default:"fid"`
 
+	// ExternalFid (UUID) column name
```

Comment:
Zou het omschrijven als `External feature id column name. When specified this ID column will be exposed to clients instead of the regular FID column. It allows one to offer a more stable IDs to clients instead of an auto-generated FIDs. External FID column should contain UUIDs.`